.github/workflows/deploy.yml

```diff
@@ -25,6 +25,24 @@ jobs:
       - name: Run ESLint
         run: npm run lint
 
+  mocha-unit-tests:
```

Comment:
Can this have it's own yaml file, similar to gajira

Reply:
updated